python/feast_spark/pyspark/launchers/k8s/k8s.py

```diff
@@ -189,6 +198,9 @@ def __init__(
             self._resource_template = _load_resource_template(resource_template_path)
         else:
             self._resource_template = yaml.safe_load(DEFAULT_JOB_TEMPLATE)
+        self._scheduled_resource_template = yaml.safe_load(
+            DEFAULT_SCHEDULED_JOB_TEMPLATE
```

Comment:
so template is not configurable?

Reply:
like how we set resources for the job?

Reply:
https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/docs/api-docs.md#sparkoperator.k8s.io/v1beta2.ScheduledSparkApplication

These are the configurable options. Unlike job template, i am not entirely sure whether there's much value in configuring the above options, so i leave it as default for the time being. If needed we can add it in the future.

Reply:
As for the specification for sparkapplication (not scheduledsparkapplication), it's still configurable as usual.